dotcom-rendering/src/server/prod-server.ts

```diff
@@ -87,34 +73,26 @@ export const prodServer = () => {
 	app.post('/KeyEvents', logRenderTime, renderKeyEvents);
 
 	// These GET's are for checking any given URL directly from PROD
-	app.get('/Article', logRenderTime, async (req: Request, res: Response) => {
-		// Eg. http://localhost:9000/Article?url=https://www.theguardian.com/commentisfree/...
-		try {
-			const url = buildUrlFromQueryParam(req);
-			const { html, ...config } = await fetch(url).then((article) =>
-				article.json(),
-			);
-
-			req.body = config;
-			return renderArticle(req, res);
-		} catch (error) {
-			// eslint-disable-next-line no-console
-			console.error(error);
-		}
-	});
+	app.get(
+		'/Article',
+		[logRenderTime, getContentFromURLMiddleware],
```

Comment:
Should this be in an array, could it be `('/Article', logRenderTime, getContentFromURLMiddleware, async () ...)`? I don't know if there's a difference, I just haven't seen array middleware syntax in express before

Reply:
The [express docs](https://expressjs.com/en/guide/using-middleware.html#middleware.application) suggest it should be an array.

Their example:

```js
const logStuff = [logOriginalUrl, logMethod]
app.get('/user/:id', logStuff, (req, res, next) => {
  res.send('User Info')
})
```

Reply:
Taking the line from the docs:

> Middleware can also be declared in an array for reusability.

Interestingly we've not made that abstraction (and I don't think we should), that just makes me think it's not recommended, just useful, as both ways are used in the docs.

Just a wondering though.

Reply:
I think you're both right, it's nicer without the array. Done!